### ISSUE TITLE
Code for training and dataset

### ISSUE THREAD

**Issue by dldl3D** [KexianHust/Structure-Guided-Ranking-Loss#1]
Hi,

Thanks for your great work. Do you plan to release the training code and the training data processing pipeline?

**Comment by KexianHust** [comment]
Sorry for the late reply.  You can contact Ke Xian (kexian@hust.edu.cn) if you're interested in the dataset. For now, we have no plan to release the training code  as well as the data processing pipeline.
